build.gradle

```diff
@@ -5,8 +5,8 @@ buildscript {
         minSdkVersion = 21
         targetSdkVersion = 29
 
-        versionCode = System.getenv("BUILD_NUMBER") as Integer ?: 1
-        versionName = "1.0.0"
+        versionCode = System.getenv("GITHUB_RUN_NUMBER") as Integer ?: 26
+        versionName = "1.0.23"
```

Comment:
Lets leave the versionCode = 1 and versionName = 1.0.0 as a defoult values and apply the GITHUB_RUN_NUMBER also on the versionName property. Something like "1.0.${System.getenv("GITHUB_RUN_NUMBER")}"

Reply:
Done.